src/rpc/channel.rs

```diff
@@ -126,7 +126,7 @@ impl PendingSubscriptionSink {
         );
         let success = response.is_success();
 
-        // TODO: #1052
+        // TODO(elmattic): https://github.com/ChainSafe/forest/issues/4188
```

Comment:
Given the outcome of https://github.com/ChainSafe/forest/issues/4188, let's just remove it.